demos_community.yaml

```
@@ -1,3 +1,17 @@
+- title: Subspace Search Variational Quantum Eigensolver
+  author: Shah Ishman Mohtashim, Turbasu Chatterjee, Arnav Das
+  date: 08/23/2021
+  code: https://github.com/LegacYFTw/SSVQE
+  paper: https://arxiv.org/abs/1810.09434
+  color: heavy-rain-gradient
+  description: |
+    The variational quantum eigensolver (VQE) is an algorithm for searching the
+    ground state of a quantum system. The SSVQE uses a simple technique to find
+    the excited energy states by transforming the |0⋯0⟩ to the ground state,
+    and another orthogonal basis state |0⋯1⟩ to the first excited state and so on.
+    As a demonstration, the weighted SSVQE is used to find out the excited states
+    of a Transverse Ising model with 4 spins and that of the Hydrogen molecule.

```

```suggestion
    of a transverse Ising model with 4 spins and that of the hydrogen molecule.
```
I am not an English expert but googling shows that they are usually lower case...